src/rules/noImplicitDependenciesRule.ts

```diff
@@ -108,6 +108,17 @@ function walk(ctx: Lint.WalkContext<Options>) {
         }
         return dependencies.has(module);
     }
+
+    function isInWhiteList(name: string): boolean {
+        const parts = name.split(/\//g);
+        if (name[0] !== "@") {
+            return whitelist.has(parts[0]);
+        }
+        if (!whitelist.has(`${parts[0]}/${parts[1]}`) && !whitelist.has(`${parts[0]}`)) {
```

Comment:
can you reuse `getPackageName()` for all this logic somehow? it's quite duplicated.

Reply:
I don’t quite understand. When you said ‘custom paths config` you mean the import statement `import file1 from ‘@src/folder/file1’` or the the whitelist array on talent.json `[true, “dev”, [“@src”]]`? 

And about `@one/two` in NPM-speaking meaning are there anything I can read on?

Reply:
Do you mean change whitelist string “@src” to “#src” to represent scope “@src”?

Reply:
So the solution here is not to add `@src` to `paths` in tsconfig.json. Then it’s no point in merging this pull request, I saw the issue and think since it’s accepting prs then there are reasons for the issue reporter’s tsconfig setup, and I should preserve this setup. Then the comment down in the issue is the solution, and I think probably that issue should be close.

And I notice one of my prs didn’t close the it’s relating issues automatically. Is it mean to be so?